graphics/vtk/Portfile

```diff
@@ -26,20 +27,22 @@ master_sites        http://www.vtk.org/files/release/${branch}
 
 distname            VTK-${version}
 
-checksums           rmd160  bfc1baf925ba1f497f14a4576fd0a7956ee42d29 \
-                    sha256  2d5cdd048540144d821715c718932591418bb48f5b6bb19becdae62339efa75a
+checksums           sha256 71a09b4340f0a9c58559fe946dc745ab68a866cf20636a41d97b6046cb736324
```

Comment:
Please add the (updated) `rmd160` checksum that you removed. Also add the `size`.

Reply:
Yup. Those are mandatory these days. (`port lint` should now complain about them, which is another good reason to always run `port lint`.)

Reply:
Actually the `port lint` command does not give any warning or error since I have checksums (but with only sha256). 
Pushed the correct checksums btw! :)